src/elements/OcProgress.vue

```diff
@@ -35,11 +32,40 @@ export default {
       type: Number,
       required: true,
     },
+    /**
+     * The size of the progress bar.
+     * Can be `default` or `small`
+     */
+    size: {
+      type: String,
+      required: false,
+      default: "default"
+    },
+    /**
+     * The variation of the progress bar.
+     * Can be `primary` or `warning`
+     */
+    variation: {
+      type: String,
+      required: false,
+      default: "primary"
```

Comment:
please add validation for allowed variations

Reply:
Done :wink: